webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -153,6 +154,16 @@ public ScrambleRequest(){}
     public String title;
     public boolean fmc;
     public HashMap<String, Color> colorScheme;
+    public int totalAttempt;
+    public int attempt;
+    public DateTime roundStartTime;
+    
+    // totalAttempt and attempt are useful for when we have multiple attempts split in the schedule.
+    // When not using ordered scrambles, we create an array of ScrambleRequest and
+    // the position in the array is related to the attempt, hence we can print attempt number (mainly for FMC and MBLD)
+    // and use, for example, scramble 1 or 3 (3 here is array.length)
+    // But when using ordered scrambles, each attempt is treated like an individual ScrambleRequest.
+    // We use totalAttempt as array.length and attempt as the position that scramble would be on the array.
```

Comment:
Thanks for adding this comment. Could you move things around so they're visibly "attached" to `totalAttempt` and `attempt`? I'm thinking something like this:

```java
...
public HashMap<String, Color> colorScheme;

// BIG COMMENT HERE
// ...
public int totalAttempt;
public int attempt;

...
```